cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaCruiseControlConfig.java

```diff
@@ -1149,38 +1148,14 @@ private void sanityCheckSamplingPeriod() {
   }
 
   /**
-   * Sanity check for
-   * (1) {@link KafkaCruiseControlConfig#REPLICA_MOVEMENT_STRATEGIES_CONFIG} is non-empty.
-   * (2) If any strategy extends from {@link com.linkedin.kafka.cruisecontrol.executor.strategy.AbstractReplicaMovementStrategy},
-   *     then all the strategies should extend from this abstract class.
+   * Sanity check to ensure that {@link KafkaCruiseControlConfig#REPLICA_MOVEMENT_STRATEGIES_CONFIG} is non-empty.
```

Comment:
Maybe we can remove this sanity check as well -- since we have the base strategy